packages/checkbox-react/src/Checkbox.tsx

```diff
@@ -1,25 +1,28 @@
-import React, { ReactNode } from "react";
+import React, { ReactNode, ChangeEvent } from "react";
 
 interface Props {
-    children: ReactNode;
+    children?: ReactNode;
+    name: string;
     checked?: boolean;
-    onChange?: (checked: boolean) => void;
+    inline?: boolean;
+    onChange?: (name: string, checked: boolean) => void;
     className?: string;
 }
 
-export function Checkbox({ children, checked, onChange, className = "" }: Props) {
-    const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => onChange && onChange(e.target.checked);
+export function Checkbox({ children, name, checked, onChange, className = "", inline = true }: Props) {
+    const handleChange = (e: ChangeEvent<HTMLInputElement>) => onChange && onChange(name, e.target.checked);
     return (
-        <label className={`jkl-checkbox ${className}`}>
+        <label className={`jkl-checkbox ${inline ? "jkl-checkbox--inline" : ""} ${className}`}>
             <input
                 className="jkl-checkbox__input"
                 data-testid="jkl-checkbox-input"
                 checked={checked}
                 type="checkbox"
+                name={name}
                 onChange={handleChange}
             />
             <span className="jkl-checkbox__check-mark" />
-            {children}
+            {children ? children : name}
```

Comment:
This feels like a weird api:
```
<Checkbox name="Will this be displayed?" />
            <Checkbox name="Will this be displayed">Or this be displayed?</Checkbox>
```

Reply:
Yeah, that was mostly there to accommodate a string array of options in CheckboxGroup in addition to a more HTML-like implementation. Should probably be reverted.

Reply:
Should this still be here? I'm not a huge fan of this either.

Reply:
Oops, missed that one!